.github/workflows/verify.yml

```diff
@@ -10,11 +10,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2.3.4
+      - uses: actions/checkout@v2.4.0
       - name: Set up JDK 8
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v2.3.1
         with:
-          distribution: adopt
+          distribution: temurin
```

Comment:
We could probably also already use java 11 for compilation. 

I dont feel like we should support the remaining 15 percent. I would also be okay with actually switching to java 16 which would also allow you to use instance pattern matching and other nice stuff.

11 is a instant go I think. We should discuss the java 16 change together. But FAWE and World Edit are both switching to java 16 soon and will push this anyway so we could just jump on the train here. What do you think @SirYwell 

![image](https://user-images.githubusercontent.com/46890129/141100252-e8ba56d8-ffba-453b-be56-aae8035a27fb.png)

Reply:
Java 11 update is in progress for another pr, _instant_ go indeed :D Java _17_ with 1.18 would be probably a good future goal.

Reply:
It java 17 for 1.18 already confirmed?

Reply:
I think it is planned at least, Java 17 will be an LTS contrary to Java 16 and Fabric has already upgraded in its snapshot versions.